remo/reports/api/views.py

```diff
@@ -19,3 +30,77 @@ def retrieve(self, request, pk):
         serializer = ActivitiesDetailedSerializer(report,
                                                   context={'request': request})
         return Response(serializer.data)
+
+
+class ActivitiesKPIFilter(django_filters.FilterSet):
+    """Filter for activities KPI endpoint."""
+    category = django_filters.CharFilter(name='functional_areas__name')
+    initiative = django_filters.CharFilter(name='campaign__name')
+
+    class Meta:
+        model = NGReport
+        fields = ['country']
+
+
+class ActivitiesKPIView(APIView):
+
+    def get(self, request):
+        """Returns serialized data for Activities KPI"""
+
+        queryset = NGReport.objects.filter(report_date__lte=now())
```

Comment:
Here [1] specs say that ``total`` means ``to date``. I asssume that we are excluding future activities.

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1144188#c0